project.godot

```diff
@@ -25,7 +25,7 @@ singletons=[ "res://addons/godot-openvr/godot_openvr.gdnlib" ]
 
 [physics]
 
-common/physics_fps=90
+common/physics_fps=144
```

Comment:
Got to love the added smoothness, I suppose :)

Reply:
mostly because I am testing on a quest, you can change it depending on the refresh rate of your headset (maybe one day we could had a way to change it easily)